myapp/routes.go

```diff
@@ -37,6 +38,7 @@ func (a *application) routes() *chi.Mux {
 		r.Put("/update-profile/{profileID:[0-9]+}", a.Handlers.UpdateProfile)
 	})
 
+	a.App.Routes.Get("/spotauth/callback", a.Handlers.SpotifyAuthorizationCallback)
 	a.App.Routes.Get("/spotauth/callback", a.Handlers.SpotifyAuthorizationCallback)
```

Comment:
I don't think we want to be capturing this twice

Reply:
Ah, whoops. That was a mistake. New pr coming right now.